shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -35,12 +41,82 @@ class AccessibilityBridge;
  */
 @property(nonatomic, strong) SemanticsObject* parent;
 
+/**
+ * The accessibility bridge that this semantics object is attached to. This
+ * object may use the bridge to access contextual application information.
+ */
+@property(nonatomic, readonly) shell::AccessibilityBridge* bridge;
+
+/**
+ * The accessibility node used to produce this semantics object.
+ */
+@property(nonatomic, readonly) blink::SemanticsNode node;
+
+/**
+ * Updates this semantics object using data from the `node` argument.
+ */
+- (void)setSemanticsNode:(const blink::SemanticsNode*)node NS_REQUIRES_SUPER;
+
+/**
+ * Whether this semantics object has child semantics objects.
+ */
+@property(nonatomic, readonly) BOOL hasChildren;
+
+/**
+ * Direct children of this semantics object. Each child's `parent` property must
+ * be equal to this object.
+ */
+@property(nonatomic, readonly) std::vector<SemanticsObject*>* children;
+
+- (BOOL)willCauseLayoutChange:(const blink::SemanticsNode*)node;
```

Comment:
This method name is not idiomatic Objective-C. `-(BOOL)nodeWillCauseLayoutChange:...` works better.